src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -7071,11 +7071,15 @@ AlterExpression AlterExpression():
                 )
                 |
                 LOOKAHEAD(3) (
-                    ( LOOKAHEAD(2) <K_COLUMN> { alterExp.hasColumn(true); } )?
+                    (
+                        <K_COLUMN> { alterExp.hasColumn(true); }
+                        |
+                        <K_COLUMNS> { alterExp.hasColumns(true); }
```

Comment:
It seems I auto-formatted this file. This is the only change, but I can remove the formatting bit if that matters.